apps/examples/ble_perfs/README.txt

```diff
@@ -0,0 +1,8 @@
+examples/ble_rmc
+^^^^^^^^^^^^^^
+
+  This is BLE remote controller example.
```

Comment:
Please check the description.
Isn't it the performance test?

Reply:
done